apps/shopifypartners/shopify_partners.star

```diff
@@ -0,0 +1,375 @@
+"""
+
+APP_ID = "shopify_sales"
+
+def api_fetch(partnermetricsCookie, request_config):
+    cache_key = "{}/{}".format(APP_ID, base64.encode(json.encode(request_config)))
```

Comment:
Since `request_config` only holds the `now` field, which is set to today's date, this cache key will be the same for all users of the app. I think you'll want to include the `partnerMetricsCookie` in the key as well, to avoid one user reading another users data?

Reply:
added `partnerMetricsCookie` to `cache_key`